### DISCUSSION TITLE
Visual Basic in Visual Studio 2022

### DISCUSSION THREAD

**Discussion by rbr146** [dotnet/winforms#7731] [Q&A] [answered]
Hi,
I have inserted many controls on a form and i saved it. But now when i try to open the project i get this error joined.
Could someone help ? The MSDN gives a oops error message. I have no possibility to correct this error (i have no idea about and no clear message and no line number...)

Many thanks for help
René rbr146@yahoo.fr
![ErreurVisualStudio](https://user-images.githubusercontent.com/48650267/188204282-f37295da-68b0-43b6-9b3f-99c18de8ed88.PNG)

**Comment by rbr146** [comment]
MSDN gives this link : https://go.microsoft.com/fwlink/?LinkId=616237 
but it gives the oops message

**Comment by KlausLoeffelmann** [comment, marked as answer]
Is this still an issue with the current version of the WinForms Designer in VB?
Also, please take a look at the recent changes around VB and the Application Framework:

https://devblogs.microsoft.com/dotnet/category/visual-basic/

If this is still a problem, please open a bug report in the WinForms repo. Thanks!

**Reply by RussKie** [reply]
https://aka.ms/winforms/designer/troubleshooting
